cmake/template.pc.in

```diff
@@ -0,0 +1,6 @@
+Name: @PROJECT_NAME@
+Description: @PROJECT_DESCRIPTION@
+Version: @PROJECT_VERSION@
+
+Libs: -L@CMAKE_INSTALL_FULL_LIBDIR@
+Cflags: -I@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Comment:
This one I am not really happy with. It should include the `-l` flags and `-D` defines, but I couldn't find a way to design with `file(GENERATE)`